Okta.AspNet/OktaMvcOptions.cs

```diff
@@ -44,6 +44,12 @@ public class OktaMvcOptions : Abstractions.OktaWebOptions
         /// <value>The scope.</value>
         public IList<string> Scope { get; set; } = OktaDefaults.Scope;
 
+        /// <summary>
+        /// Gets or sets the <see cref="LoginMode"/>.
```

Comment:
I would elaborate here, maybe something like "Controls the login behavior to redirect to internal or Okta-hosted login"